packages/gnome_session.rb

```diff
@@ -0,0 +1,50 @@
+# Adapted from Arch Linux gnome-session PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/gnome-session/trunk/PKGBUILD
+
+require 'package'
+
+class Gnome_session < Package
+  description 'The GNOME Session Handler'
+  homepage 'https://gitlab.gnome.org/GNOME/gnome-session'
+  @_ver = '40.beta'
+  version @_ver
+  compatibility 'x86_64 , armv7l'
```

Comment:
Missing aarch64?

Reply:
Ah yes, I need to modify my script to add those...